package.json

```diff
@@ -31,7 +31,7 @@
     "lerna": "3.13.1",
     "ts-node": "8.0.3",
     "tslint": "5.14.0",
-    "typescript": "3.3.3333",
+    "typescript": "3.3.4000",
```

Comment:
@wtrocki I can test both of these PRs.

Reply:
No need to test it really. The build is passing so there's obviously nothing broken by the typescript update 😄